superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/ColumnSelect.tsx

```diff
@@ -98,17 +98,17 @@ export function ColumnSelect({
     }
     if (datasetId != null) {
       cachedSupersetGet({
-        endpoint: `/api/v1/dataset/${datasetId}`,
+        endpoint: `/api/v1/dataset/${datasetId}/column`,
```

Comment:
```suggestion
        endpoint: `/api/v1/dataset/${datasetId}/columns`,
```

Should this end point be `/columns`?

Reply:
@ktmud and @michael-s-molina I agree that it should be `columns` and not `column`—it should also be `datasets` and not `dataset`, however per,

<img width="1151" alt="Screenshot 2022-12-09 at 6 53 13 AM" src="https://user-images.githubusercontent.com/4567245/206527505-da57eb7e-6553-4c55-86f8-6deeece06a53.png">

we already use the singular form and thus `column` ensures consistency which I sense is more important.